src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -943,4 +952,46 @@ public void release()
             }
         }
     }
+
+    public static class Stats {
+        private final Clock clock;
+
+        public Stats(Clock clock)
+        {
+            this.clock = clock;
+        }
+
+        public Stats()
+        {
+            this(Clock.systemUTC());
+        }
+
+        /**
+         * An average of the time it takes to make allocate channel requests
+         * to JVB.
+         */
+        private final RateStatistics allocateChannelsReqTimes =
+            new RateStatistics((int)Duration.ofMinutes(1).toMillis());
+
+        private final RateStatistics healthCheckReqTimes =
```

Comment:
This is not used anywhere?

Reply:
Yeah, I had started this but wasn't sure how/if we wanted to measure it because of the retry stuff...I can either remove it or we can come up with an idea of how we want to do the health check tracking (though with something other than `RateStatistics`, as you pointed out).